package.json

```diff
@@ -38,11 +38,10 @@
     "select2": "4.0.13"
   },
   "devDependencies": {
-    "@automattic/calypso-build": "6.5.0",
-    "@automattic/puppeteer-utils": "github:Automattic/puppeteer-utils",
```

Comment:
`@automattic/puppeteer-utils ` was uninstalled.

Since we disabled the e2e tests, for now, it's not a big deal. When we reactivate it, we should find another solution for this. See more details about the issue we had and why we shouldn't continue using this dependency: https://github.com/Automattic/puppeteer-utils/issues/8#issuecomment-848402804